nearlib/near.js

```diff
@@ -201,6 +201,18 @@ class Near {
                 alreadyDisplayedLogs.push(line);
             }
             if (result.status == 'Completed') {
+                for (j = result.logs.length - 1; j >= 0; --j) {
+                    let r = result.logs[j];
+                    if (r.result && r.result.length > 0) {
+                        result.lastResultUnparsed = r.result;
+                        try {
+                            result.lastResult = JSON.parse(Buffer.from(r.result).toString());
+                        } catch (e) {
+                            // can't parse
```

Comment:
Should we log the error?

Reply:
For non-AS JSON results, we can still parse raw data. E.g. using lastResultUnparsed

Reply:
Okay